libs/auth-api-lib/src/lib/resources/admin/admin-scope.service.ts

```diff
@@ -113,10 +113,34 @@ export class AdminScopeService {
       throw new BadRequestException(`Scope name "${input.name}" already exists`)
     }
 
+    // We need to manually validate displayName and description, since we reuse the same dto for both create and publish
+    const translatedValuesErrorMsg =
+      'Scope displayName and description are required'
+
+    if (!input.displayName || !input.description) {
+      throw new BadRequestException(translatedValuesErrorMsg)
+    }
```

Comment:
Don't understand why this isn't validated like other fields?

Reply:
The fields are optional code wise because we extend the AdminPatchScopeDto, but required in create in the real world.

```
export class AdminCreateScopeDto extends AdminPatchScopeDto {
  @IsString()
  @IsNotEmpty()
  @ApiProperty({ example: '@island.is' })
  name!: string
}
```